rspack.config.ts

```diff
@@ -10,7 +10,7 @@ import type {
 } from '@rspack/core';
 import rspack from '@rspack/core';
 import ReactRefreshRspackPlugin from '@rspack/plugin-react-refresh';
-import {sentryWebpackPlugin} from '@sentry/webpack-plugin';
+import {sentryWebpackPlugin} from '@sentry/webpack-plugin/webpack5';
```

Comment:
necessary to completely remove webpack dependency. otherwise the webpack plugin attempts to check version by importing webpack